src/GitHub.Exports/Settings/IPackageSettings.tt

```diff
@@ -0,0 +1,31 @@
+﻿<#@ template debug="false" hostspecific="true" language="C#" #>
+<#@ assembly name="System.Core" #>
+<#@ import namespace="System.Linq" #>
+<#@ import namespace="System.Text" #>
+<#@ import namespace="System.Collections.Generic" #>
+<#@ import namespace="System.IO" #>
+<#@ assembly name="$(PackageDir)\Newtonsoft.Json.6.0.8\lib\net45\Newtonsoft.Json.dll" #>
+<#@ import namespace="Newtonsoft.Json.Linq" #>
+<#@ output extension=".cs" #>
+<#
+var file = this.Host.ResolvePath(@"..\..\common\settings.json");
+var json = JObject.Parse(File.ReadAllText(file));
+#>
+// This is an automatically generated file, based on settings.json and PackageSettingsGen.tt
+/* settings.json content:
+<#@ include file="..\..\common\settings.json" #>
+*/
+namespace GitHub.Settings
+{
+    public interface IPackageSettings
+    {
+        void Save();
+<#
+foreach (var j in json["settings"].Children()) {
+#>
+        <#= j["type"] #> <#= j["name"] #> { get; set; }
```

Comment:
Don't worry about it. :smile: Trying to get generated code formatted correctly with TT is a time sink.